src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -774,7 +859,11 @@ private function replacePlaceholders(Tokens $tokens, string $alignStrategy, stri
             foreach ($lines as $index => $line) {
                 if (substr_count($line, $placeholder) > 0) {
                     $groups[$groupIndex][] = $index;
-                } elseif ('=>' !== $tokenContent) {
+                } elseif (
```

Comment:
The specific behavior won't be hardcoded for `=>` but instead will be an option `BY_SCOPE` for alignment.